### DISCUSSION TITLE
Restoring a Failed Manager

### DISCUSSION THREAD

**Discussion by asrachid** [Security-Onion-Solutions/securityonion#13713] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

20

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Could you please help me with detailed steps to do a restore of a failed manager? 
I am in a distributed architecture, my manager is backed up daily to a remote locations so all the following files are available:
/etc/pki, /etc/salt, /opt/so/saltstack/local and /nsm/kratos.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
Please take a look at this discussion and see if it helps:
https://github.com/Security-Onion-Solutions/securityonion/discussions/13512
